### ISSUE TITLE
graphics/{index}_gl_support_{product_slug} false alarm in nvlink case.

### ISSUE THREAD

**Issue by os369510** [canonical/checkbox#357]
### Bug Description

In workstation, if users use nvlink (two same nvidia graphic cards attached on a system and connecting with nvlink).
The graphics/{index}_gl_support_{product_slug} will report failed.
In this scenario, the one if for display, the other one is for computing.

### To Reproduce

In my environment:

1. DISPLAY=:0 checkbox-cli run com.canonical.certification::graphics_card com.canonical.certification::graphics/1_gl_support_GA102GL__RTX_A6000_

### Environment

OS: Ubuntu 22.04 LTS "Jammy Jellyfish"
In a Lenovo workstation (without iGPU):
0000:27:00.0 VGA compatible controller [0300]: NVIDIA Corporation GA102GL [RTX A6000] [10de:2230] (rev a1) (prog-if 00 [VGA controller])
0000:38:00.0 VGA compatible controller [0300]: NVIDIA Corporation GA102GL [RTX A6000] [10de:2230] (rev a1) (prog-if 00 [VGA controller])

### Relevant log output

```shell
$ DISPLAY=:0 checkbox-cli run com.canonical.certification::graphics_card 

com.canonical.certification::graphics/1_gl_support_GA102GL__RTX_A6000_
Using sideloaded provider: plainbox-provider-pc-sanity, version 1.20.0.dev0 from /var/tmp/checkbox-providers/plainbox-provider-pc-sanity
===========================[ Running Selected Jobs ]============================
--------------[ Running job 1 / 1. Estimated time left: 0:00:00 ]---------------
------[ Generate an entry for each graphics card present in the system. ]-------
ID: com.canonical.certification::graphics_card
Category: com.canonical.plainbox::uncategorised
... 8< -------------------------------------------------------------------------
bus: pci
category: VIDEO
driver: nvidia
gpu_count: 2
index: 1
path: /devices/pci0000:26/0000:26:01.0/0000:27:00.0
prime_gpu_offload: Off
product: GA102GL [RTX A6000]
product_id: 8752
product_slug: GA102GL__RTX_A6000_
subproduct_id: 5209
subvendor_id: 6058
switch_to_cmd: prime-select intel
vendor: NVIDIA Corporation
vendor_id: 4318
vendor_slug: NVIDIA_Corporation

bus: pci
category: VIDEO
driver: nvidia
gpu_count: 2
index: 2
path: /devices/pci0000:37/0000:37:01.0/0000:38:00.0
prime_gpu_offload: Off
product: GA102GL [RTX A6000]
product_id: 8752
product_slug: GA102GL__RTX_A6000_
subproduct_id: 5209
subvendor_id: 6058
switch_to_cmd: prime-select nvidia
vendor: NVIDIA Corporation
vendor_id: 4318
vendor_slug: NVIDIA_Corporation

------------------------------------------------------------------------- >8 ---
Outcome: job passed
--------------[ Running job 1 / 1. Estimated time left: 0:00:00 ]---------------
-------[ Test OpenGL support for NVIDIA Corporation GA102GL [RTX A6000] ]-------
ID: com.canonical.certification::graphics/1_gl_support_GA102GL__RTX_A6000_
Category: com.canonical.plainbox::graphics
... 8< -------------------------------------------------------------------------
Authorization required, but no authorization protocol specified
Error: unable to open display
------------------------------------------------------------------------- >8 ---
Outcome: job failed
Finalizing session that hasn't been submitted anywhere: checkbox-run-2023-03-09T05.25.05
==================================[ Results ]===================================
 ☑ : Generate an entry for each graphics card present in the system.
 ☒ : Test OpenGL support for NVIDIA Corporation GA102GL [RTX A6000]
```

```
$ grep "" /sys/class/drm/*/status
/sys/class/drm/card0-DP-1/status:disconnected
/sys/class/drm/card0-DP-2/status:connected
/sys/class/drm/card0-DP-3/status:disconnected
/sys/class/drm/card0-DP-4/status:disconnected
/sys/class/drm/card1-DP-5/status:disconnected
/sys/class/drm/card1-DP-6/status:disconnected
/sys/class/drm/card1-DP-7/status:disconnected
/sys/class/drm/card1-DP-8/status:disconnected
```
```


### Additional context

Suggest to use `nvidia-smi nvlink --checkBridge` to check the nvlnk status when collecting device information.

With nvlink:
```
$ nvidia-smi nvlink --checkBridge
GPU 0: NVIDIA RTX A6000 (UUID: GPU-994cd4b9-a26a-10cb-4d1f-78698c8ededd)
GPU 1: NVIDIA RTX A6000 (UUID: GPU-4b3de229-e3da-a57a-f067-0d6f9b9b63c8)
```

Without nvlink:
```
$ nvidia-smi nvlink --checkBridge
```
